sites/svelte-5-preview/src/routes/svelte/[...path]/+server.js

```diff
@@ -0,0 +1,31 @@
+import compiler_cjs from '../../../../../../packages/svelte/compiler.cjs?url';
+import package_json from '../../../../../../packages/svelte/package.json?url';
+import { read } from '$app/server';
+
+const files = import.meta.glob('../../../../../../packages/svelte/src/**/*.js', {
+	eager: true,
+	as: 'url'
+});
+
+// service worker requests files under this path to load the compiler and runtime
+export async function GET({ params }) {
+	let url = '';
+	if (params.path === 'compiler.cjs') {
+		url = compiler_cjs;
+	} else if (params.path === 'package.json') {
+		url = package_json;
+	} else {
+		const path = '../../../../../../packages/svelte/' + params.path;
+		url = files[path];
+	}
+
+	const response = read(url);
+	return new Response(response.body, {
+		status: response.status,
+		statusText: response.statusText,
+		headers: {
+			...response.headers,
+			'Cache-Control': 'public, max-age=30' // 30 seconds so that redeploys are picked up quickly
```

Comment:
this seems slightly dangerous — you could imagine a situation where one module was read from cache and another was read from server because they straddled the line, and they belong to different versions. realistically i'm not sure a 30 second max-age really buys us that much anyway.

it would be good if we could keep just using static files for this (so people get 304s where appropriate, etc). maybe there's another way?

Reply:
What about removing the cache but prerender this?

Reply:
Yeah, that makes sense

Reply:
made the change